### ISSUE TITLE
New Tweeters Volunteer: @Soteria-Smart-Contracts aka Wedergarten

### ISSUE THREAD

**Issue by IstoraMandiri** [ethereumclassic/volunteer#89]
Following up on https://github.com/ethereumclassic/tweets-eth_classic/issues/19, this issue is to document the on-boarding process of a new user to the @ethereumclassic/tweeters team.

The new on-boarding process requires some [checks](https://github.com/ethereumclassic/volunteer#on-boarding) to help keep the org secure.

This issue is open to everyone including the new volunteer to provide additional details or objections before existing volunteers make a recommendation to @ethereumclassic org admins to add as a volunteer to help manage tweets.

| Volunteer | @Soteria-Smart-Contracts |
|---|---|
| 2FA | Unknown |
| Contributions | ETC Calls, Discord, Contract Deployments |
| Anti-Sybil | [Pass](https://www.youtube.com/watch?v=zvL9hu46kZQ) |

@Soteria-Smart-Contracts, I'd prefer if you are contributing via your personal, rather than company Github account, do you have one, or could you make one for @Wedergarten (currently not taken so snap it up!)?

**Comment by Soteria-Smart-Contracts** [comment]
Yeah for sure, give me a few hours ill come back with my new account.

**Comment by IstoraMandiri** [comment]
@Soteria-Smart-Contracts did you have an update on this? As an active member i'd like to have you help with tweet approvals.

**Comment by Soteria-Smart-Contracts** [comment]
Not switching between accounts, happy to help from this one, up to you

**Comment by IstoraMandiri** [comment]
All right with me, unless there are objections, @realcodywburns could you help with adding him to the  (cc) @ethereumclassic/tweeters team? I believe he first needs to be added to the org.

**Comment by Soteria-Smart-Contracts** [comment]
Hey, still down to be in. Would like to review tweets since some with typos seem to be slipping through.

**Comment by IstoraMandiri** [comment]
cc @realcodywburns

**Comment by realcodywburns** [comment]
no objections. Will update by 9/8
